### PR TITLE
#167328280 improve the login feature

### PR DESCRIPTION
**Pull request by Kabalisa**
Description
=======
This chore is for improving the login feature. previously, a user was able to go back to log in while they are logged in and still authenticated.

Type of change
=======
-redirect a logged-in user to articles page
- remove an expired token from the local storage

How has it been tested
=======
- pull this branch
- run ```npm install```
- run ```npm start``` to start our react app
-login into authorshaven
-try going back to login

Checklist:
=======
N/A

Pivotal tracker story ID
=======
[#167328280](https://www.pivotaltracker.com/story/show/167328280)

### CODE REVIEW

File: src/components/authentication/login.jsx [hunk 1]
```diff
@@ -70,33 +69,44 @@ componentDidMount(){
...
-        localStorage.setItem('token', token);
-      }
+      Helpers.setToken(token);
     }
-    isAuthanticated && this.props.history.push('/articles');
+    isAuthanticated && this.props.history.replace('/articles');
   }
   render() {
-
+    console.log('in login', this.props);
```
Comment (AJAkimana) [inline src/components/authentication/login.jsx]: Remove this console.

**Review by AJAkimana — COMMENTED** [review]
Great work, but remove the console in login.jsx file

**Review by ericnyirimana — COMMENTED** [review]
Rebase your PR @Kabalisa

**Review by AJAkimana — APPROVED** [review]
Good work @Kabalisa

**Review by ericnyirimana — APPROVED** [review]

**Pull request merged**